course3/homework-string/build.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
```

Comment:
You shouldn't set compiler in CMake, that CMake's job to figure that out.

You need to run cmake to generate makefile (or ninja file, Visual Studio solution etc etc), but usually you do it only once.

On Windows - where things tend to get unnecessary complex - you may have to use correct "prompt" - a shell instance with some environment variables set. MinGW should come with one.

Using cmake should be a simple thing.